translations/translations.json

```
@@ -6357,7 +6357,8 @@
       "pageTitle": "Lokale Informationen für Dich",
       "metaDescription": "{{appName}} ist Dein digitaler Guide für Deutschland. Finde lokale Informationen, Veranstaltungen und Beratung. Immer aktuell und in Deiner Sprache.",
       "cityNotFound": "Deine Region nicht gefunden? Werde jetzt aktiv!",
-      "suggestToRegion": "{{appName}} in Deiner Region vorschlagen"
+      "suggestToRegion": "{{appName}} in Deiner Region vorschlagen",
+      "searchCityTip": "Wählen Sie Ihren Bezirk oder die nächstgrößere Stadt, z. B. Landkreis München"
```

🔧 We use `du` instead of `Sie` and `Region` over `Bezirk`:
```suggestion
      "searchCityTip": "Wähle deine Region oder die nächstgrößere Stadt, z. B. Landkreis München"
```
🙃 Perhaps `searchCityDescription` would fit a little bit better?